lab/lab2/js/part2-app-state.js

```diff
@@ -53,7 +69,9 @@ var plotMarkers = function() {};
   user's input.
 ===================== */
 
-var removeMarkers = function() {};
+var removeMarkers = function(d) {
+  map.removeLayer(d);
```

Comment:
You need to iterate over the inputs to this function to remove each marker. (an effective one being `_.each`

something like: 

```js
_.each(markers, function(d) {
    map.removeLayer(d);
```

This is insufficient.

Reply:
Thank you, Ross. I changed this function to this:
```
var removeMarkers = function(markers) {
  _.each(markers, (d) => {
    map.removeLayer(d)
  });
};
```

Reply:
Nice. Now it works 👍